modules/bound-sa-tokens-configuring.adoc

```diff
@@ -67,13 +67,32 @@ If the output shows a message similar to one of the following messages, the upda
 ** `3 nodes are at revision 11; 0 nodes have achieved new revision 12`
 ** `2 nodes are at revision 11; 1 nodes are at revision 12`
 
-... Manually restart all pods in the cluster:
+.. Optional: Force the holder to request a new bound token either by performing a rolling node restart or by manually restarting all pods in the cluster.
++
+// TODO: When would a user NOT need to do this?
```

Comment:
it's true that very likely a user might always want to do this but I wouldn't push restarting all their pods on them in the docs